tests/requirements-libraries.txt

```diff
@@ -26,7 +26,8 @@ sphinx ; python_version != '3.3'
 
 pyzmq
 zope.interface  # Required for test_namespace_package
-numpy
+numpy ; python_version < '3.0' or python_version >= '3.4'
+numpy<1.12 ; python_version >= '3.0' and python_version < '3.4'
```

Comment:
No, numpy ended support for python 3.3. It's probably too complicated, but this way is supposed to be more "robust",